_data/coremembers.yml

```diff
@@ -47,4 +47,22 @@
   name: "Syed Abdullah",
   url: "http://syedabdullah.net",
   img: "/img/team/syed.png"
-}
\ No newline at end of file
+}
+
+- {
+  name: "Alex Fong Jie Wen",
+  url: "https://github.com/alexfjw",
+  img: "/img/team/alexfong.png"
+}
+
+- {
+  name: "Tan Jun Kiat",
+  url: "https://github.com/junkiattan",
+  img: "/img/team/junkiattan.png"
+}
+
+- {
+  name: "Lee Yan Hwa",
+  url: "https://github.com/leeyh20",
+  img: "/img/team/yanhwa.jpg"
+}
```

Comment:
You can move these guys to the 'Past members' section: Min Oo, Hanyang, Yu Ting, Victor, Shunhao